util.py

```diff
@@ -127,6 +148,12 @@ def haversine_distance(lat1, lon1, lat2, lon2):
 
     return EARTH_RADIUS_IN_FEET * c
 
+def distance(x0, y0, x1, y1):
```

Comment:
get_distance_between_points() would be more descriptive

Reply:
Maybe, but this is just straight Pythagorean distance, and `distance()` or `length()` are commonly used names for this functionality.